html/janus.js

```diff
@@ -95,6 +97,7 @@ Janus.useDefaultDependencies = function (deps) {
 		newWebSocket: function(server, proto) { return new socketCls(server, proto); },
 		extension: (deps && deps.extension) || defaultExtension,
 		isArray: function(arr) { return Array.isArray(arr); },
+		/*global adapter */
```

Comment:
imho I'd prefer adding the globals to eslint config instead of polluting the script with comments like this

Reply:
Will do.